CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 1.0.0
+
+Security and privacy improvements:
+- Faster and more secure 2-layer E2E encryption, using double ratchet algorithm with Curve448 and Curve25519 for key agreement.
```

Comment:
Elaborate on 2-layer? I.e. DR for connection level, NaCl crypto_box encryption scheme for queue level. I think it'd be appropriate technicality for changelog + it'd explain why 2 layers are implemented (remove queues correlation in preparation for queues redundancy)

Reply:
there is a link to blog post with more details.